Renew

martindurant opened the pull request:
Posting this now, as it provides a working cluster setup on a single command. 

The README should be updated to reflect the changes and how to use it.

In the future, I envisage having the worker pods being started from the scheduler rather than remotely, possibly with its own management bokeh dashboard.

mrocklin committed:
update dscheduler to dask-scheduler, etc.

mrocklin committed:
update to newer versions of dask

mrocklin committed:
add resources to worker and scheduler

Commit:
Rewrite all

Commit:
Reorder

Commit:
rener docker image

Commit:
Working implementation

martindurant commented:
Links to docker image [here](https://hub.docker.com/r/mdurant/docker-distributed/builds/b6dwi535muifvvtptvccuur/)

mrocklin commented on kubernetes/distributed.yaml:
```diff
@@ -1,73 +1,64 @@
-          image: ogrisel/distributed:latest
-          args: ["start-dscheduler.sh"]
+        - name: dask-scheduler
+          image: mdurant/docker-distributed:renew
```
I expect this docker container version to change in the future.  Would using version numbers or latest make more sense here than a name like `renew`?

martindurant replied:
Yes, after merging this branch (renew is the branch name)

mrocklin commented on scripts/start-dask-worker.sh:
```diff
@@ -0,0 +1,6 @@
+echo "Detected $NCPUS cpus"
+# Alternatively start one python process per CPU
+# exec su $BASICUSER -c "env PATH=$PATH dask-worker dask-scheduler:8786 --nthreads 1 --nprocs $NCPUS $*"
+exec su $BASICUSER -c "env PATH=$PATH dask-worker dask-scheduler:8786 $*"
```
Is this script still used?

martindurant replied:
No, it isn't called - I haven't gone through removing dead code yet.

mrocklin commented:
This probably doesn't need to be hosted on my github organization any longer.  I'd be happy for you to take it over or to move it to the dask organization.  Regardless we should probably change the repository name.

martindurant commented:
It should probably live in dask/ , even if it stays in "use if useful" status. I do want to make the cluster manager for kubernetes that would work in combination with this code.

Commit:
Update documentation

plus references to dscheduler, Executor, etc., in examples (other
functionality not tested)

Commit:
Further small readme update

Commit:
Change readme title
